web_widget_dropdown_dynamic/__init__.py

```diff
@@ -1 +1 @@
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/LGPL.html).
```

Comment:
you have to use lowercase, this link does not work.

Reply:
Thank you! Will update.